.github/workflows/ci.yml

```diff
@@ -0,0 +1,90 @@
+name: CI
+
+on: push
```

Comment:
`on: [push, pull_request]`?

Reply:
The thing is:

> If you intend to use the pull_request event to trigger CI tests, we recommend that you set up your workflow configuration to listen to the push event.

https://help.github.com/en/articles/events-that-trigger-workflows#pull-request-events-for-forked-repositories

I'm not sure what that means, and not sure if it matches the latest implementation.

Reply:
yeah i just read that too. very confusing

Reply:
you see it ran in your fork?
https://github.com/teohhanhui/api-platform-core/runs/209804563

Reply:
Yeah, I know, that's why I'm thinking of merging this into `api-platform/core` and see how it goes.

Reply:
Found this: https://stackoverflow.com/a/57621843/1529493

Reply:
I've updated accordingly. Let's see whether it triggers on PRs correctly after this is merged.